bundle-size/test/api.test.js

```diff
@@ -33,25 +33,29 @@ describe('bundle-size api', () => {
   let probot;
   let app;
   const db = dbConnect();
-  const nodeCache = new NodeCache();
+  let nodeCache;
+  let logWarnSpy;
 
-  beforeAll(async () => {
+  beforeAll(async done => {
     await setupDb(db);
 
     probot = new Probot({});
     app = probot.load(app => {
       const githubUtils = new GitHubUtils(new Octokit(), app.log, nodeCache);
       installApiRouter(app, db, githubUtils);
     });
+    // Stub app.log.warn to silence test log noise
+    logWarnSpy = jest.spyOn(app.log, 'warn').mockImplementation();
```

Comment:
Why not just set LOG_LEVEL to be higher than `warn` in that case?

Reply:
I opted for this because every single test logs a warning about the bundle size not existing, but if you had logging enabled for debugging purposes (at INFO level), you'd still be able to see all the info logs.

Reply:
Ack